include.mk

```diff
@@ -4,3 +4,6 @@ info:
 	@echo "Registry: ${DOCKER_REGISTRY}"
 	@echo "Project: ${PROJECT}"
 	@echo "Image: ${IMAGE}"
+
+TAG ?= $(shell echo `date +"%s"`_`date +"%N"`)
```

Comment:
Actually, we're going to replace this timestamp based TAG with a Github short hash in the next PR.